test_discovery/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>test_discovery</name>
+  <version>0.14.0</version>
+  <description>
+    Test discovery behaviors in ROS 2 using semiautomated tests.
+  </description>
+
+  <maintainer email="brandon@openrobotics.org">Brandon Ong</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <author email="sloretz@openrobotics.org">Shane Loretz</author>
+
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+
+  <!-- <depend>mininet</depend> TODO this rosdep key -->
+  <depend>rclcpp</depend>
+  <depend>test_msgs</depend>
+  <depend>pytest</depend>
```

Comment:
```suggestion
  <depend>ament_cmake_pytest</depend>
```

Reply:
Improved package.xml in 4df83da152087e480e9c7ce42d32d3c2fb448e48

`ament_cmake_pytest` is a `test_depend`, but `pytest` is still a run depend for the tests that run as root